napari_allencell_annotator/view/viewer.py

```diff
@@ -44,4 +58,97 @@ def alert(self, alert_msg: str) -> None:
         show_info(alert_msg)
 
     def get_layers(self) -> List[Layer]:
+        """
+        Returns a list of all layers in the viewer.
+        """
         return list(self.viewer.layers)
+
+    def get_all_points_layers(self) -> List[Points]:
+        """
+        Returns a list of all point layers in the viewer.
+        """
+        return [layer for layer in self.get_layers() if isinstance(layer, Points)]
+
+    @staticmethod
+    def order_point(point: np.ndarray, image_dims_order: str) -> Tuple:
+        """
+        Orders a point according to the image dimension and returns it as a tuple
+
+        Parameters
+        ----------
+        point: np.ndarray
+            A point in a point layer
+        image_dims_order: str
+            The dimension of the image
+
+        Returns
+        -------
+        Tuple[float]
+            A tuple containing ordered point coordinates
+        """
+        point_dict: Dict[str, np.ndarray] = {"T": point[0], "C": point[1], "Z": point[2], "Y": point[3], "X": point[4]}
```

Comment:
cant assume order of points, and cant assume that there will be 5 dimensions

Reply:
assume that the order is correct for now and remove the ordering in https://github.com/aics-int/napari-allencell-annotator/pull/90/commits/d8c46768ffb6869970a3be00beae75c0fa68e6de